lib/api/api.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+
+class Api {
+  static String baseUrl = "http://159.203.1.20:8080/v0";
```

Comment:
what's this ip? did you test this on your machine or is to connect to Christian's server?